AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -630,9 +630,12 @@ - (BOOL)shouldMeasureWithSizeRange:(ASSizeRange)constrainedSize
     return NO;
   }
   
-  if (ASHierarchyStateIncludesLayoutPending(_hierarchyState)
-      && _pendingTransitionID != ASLayoutableGetCurrentContext().transitionID) {
-    return NO;
+  if (ASHierarchyStateIncludesLayoutPending(_hierarchyState)) {
+    ASLayoutableContext context =  ASLayoutableGetCurrentContext();
+    ASDisplayNodeAssert(!ASLayoutableContextIsNull(context), @"Context must not be null for a pending layout subnode");
```

Comment:
Is this safe in production, if this case actually occurs? Should we initialize it to a blank structure and then check, Or will the context return a blank structure if it is not set?